slave-base/Dockerfile.rhel7

```diff
@@ -41,7 +41,6 @@ RUN INSTALL_PKGS="bc gettext git java-11-openjdk-headless java-1.8.0-openjdk-hea
     chmod -R 775 /var/lib/alternatives && \
     chmod -R 775 /usr/lib/jvm && \
     chmod 775 /usr/bin && \
-    chmod 775 /usr/lib/jvm-exports && \
```

Comment:
Ditto regarding RHEL 7.

Reply:
I would assume so yes, this repo could definitely use a future major cleanup.